app/Helpers/database/leaderboard.php

```diff
@@ -25,6 +26,14 @@ function SubmitLeaderboardEntry(
     if ($dbResult !== false) {
         $lbData = mysqli_fetch_assoc($dbResult);
 
+        $foundGame = Game::find($lbData['GameID']);
+        if ($foundGame?->ConsoleID && !isValidConsoleId($foundGame->ConsoleID)) {
+            $retVal['Success'] = false;
+            $retVal['Error'] = "Cannot insert the value $newEntry into leaderboard with ID: $lbID because the game's system is not yet validated.";
```

Comment:
Might want a shorter error message for this.
![image](https://github.com/RetroAchievements/RAWeb/assets/32680403/67615626-44a4-4d76-8aaa-926f12affe77)

The one for unofficial achievement unlocks is "Unofficial achievements cannot be unlocked."

Maybe "Cannot submit entry for unsupported console"

Reply:
Ignore the fact that the error message says achievement. That's a bug with the new DLL.

Reply:
No problem, updated in latest to your suggestion.